src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,195 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T item, Node<T> next) {
+            this.prev = prev;
+            this.item = item;
+            this.next = next;
+        }
+    }
+
     @Override
     public void add(T value) {
+        if (first == null) {
+            addFirst(value);
+        } else {
+            Node<T> newNode = new Node<>(last, value, null);
+            last.next = newNode;
+            last = newNode;
+            size++;
+        }
     }
 
     @Override
     public void add(T value, int index) {
+        //add first
```

Comment:
please, remove all comments 
code should be readable withoud comments
```suggestion
```

Reply:
OK. But why? With comments it's easier for me and for my colleagues to understand what each methods does ...